curvefs/src/mds/topology/topology.cpp

```diff
@@ -849,6 +863,19 @@ std::list<MetaServerIdType> TopologyImpl::GetMetaServerInPool(
     return ret;
 }
 
+uint32_t TopologyImpl::GetMetaServerNumInPool(
```

Comment:
GetMetaServerInPool function already exist and you can use directly.

Reply:
done